contracts/tokens/SecurityToken.sol

```diff
@@ -531,7 +531,7 @@ contract SecurityToken is StandardToken, DetailedERC20, ReentrancyGuard, Registr
      * @return bool success
      */
     function transferWithData(address _to, uint256 _value, bytes _data) public returns (bool success) {
-        require(_updateTransfer(msg.sender, _to, _value, _data), "Transfer invalid");
+        // require(_updateTransfer(msg.sender, _to, _value, _data), "Transfer invalid");
```

Comment:
what is the reason to comment this statement?

Reply:
sorry I forgot to recover this, it was comment off for debugging :/ updated.